plutus-core/README.md

```diff
@@ -72,6 +74,7 @@ Run `nix build -f default.nix plutus.haskell.packages.plutus-core.components.exe
 
 Run `stack install plutus-core` in your terminal being in any subfolder of `plutus`. Once the build finishes, you'll be shown the following lines:
 
+<!-- @thealmarty can't successfully install plutus-core, can someone else verify/update this? -->
```

Comment:
This needs to be removed before merging. Hopefully someone can confirm this.

Reply:
We shouldn't recommend that people use `install`-based workflows. Better to use `cabal run` or something.

Reply:
Great! Updated. Is the "Tests" section under it outdated? I can't seem to find the generator?